thesis-de/content-de.tex

```diff
@@ -1892,45 +1892,46 @@ \subsection{SSD Userstory}
-  \1 Die Ausgabe und Visualisierung in einen Diagramm, ob Balken- oder
-  Graphendiagramm, soll jeweils für jede einzelne Instanz angezeigt werden
+  \1 Die Ausgabe und Visualisierung in einem Diagramm, welches vorzugsweise in
+  einem Balken- oder Graphendiagramm für jede einzelne Instanz ausgegeben
+  werden kann
   \1 Auflistung einer Statistik in Form eines Balkendiagramms, über die
   Hersteller, welche am wahrscheinlich häufigsten in den Instanzen ausfallen
   \1 Eine Benachrichtigung, ob per Email oder andere Kommunikationswege,
-  wenn ein kritischer Schreib- oder Lesezyklus erreicht worden ist.
+  wenn ein kritischer Schreib- oder Lesezyklus erreicht worden ist
 \end{outline}
 \mr%
 
 \subsection{CPU Userstory}
-\\textbf{Name der Story:}\\ Auswertung der CPU Auslastung von jeder Instanz, um Engpässe zu
+\\textbf{Name der Story:}\\ Auswertung der CPU Auslastung jeder Instanz, um Engpässe zu
 erkennen.
-\\textbf{Beschreibung:}\\ Die CPUs der heutigen Zeit werden immer schneller und besser, von
-der Taktrate, bis hin zur Verarbeitungsgeschwindigkeit der Prozesse. Bei 
-Virtuellen Maschine, werden die CPUs jeweils pro Virtueller Maschine zugewiesen,
-so kann eine Maschine 2 CPU Kerne mit z.B. 2,5 GHz Takrate zugewiesen werden.
-Auf jeder Virtuellen Maschine laufen andere Dienste, weshalb eine VM
-unterschiedliche CPUs zugewiesen haben. Manchmal kommt es hier vor, das eine VM
-zu viel Rechenleistung, bedingt durch ausgeführte Dienste benötigt. Wenn eine
-VM mehr Rechenleistung benötigt, als zugewiesen ist, kann es hier zu
-Verzögerungen kommen. Um dies zu erkennen, benötigt der Administrator, eine
-detailierte Übersicht in Form eines Graphen, wieviel CPU Leistung von einer
-Virtuellen Instanz verwendet wird, um hier einen Engpass bereits vorab zu
-erkennen und der VM mehr CPU Leistung zur Verfügung stellen zu können.
+\\textbf{Beschreibung:}\\ Die CPUs der heutigen Zeit werden in der Taktrate und
+Verarbeitungsgeschwindigkeit der Prozesse immer schneller und besser. Bei
+virtuellen Maschine, werden die CPUs jeweils pro Instanz zugeteilt, so kann eine
+VM 2 CPU Kerne mit z.B. 2,5 GHz Takrate zugewiesen haben. Auf jeder virtuellen
+Maschine laufen andere Dienste, weshalb jede VM unterschiedliche CPUs zugeteilt
+haben kann. Hier kann es vorkommen, dass eine VM mehr CPU Leistung benötigt, als
+ihr selber zugewiesen ist. Um dies frühstmöglich zu erkennen, benötigt der
+Administrator, eine detailierte Übersicht in Form eines Graphen, wieviel CPU
+Leistung von einer virtuellen Instanz verwendet und vorhanden ist. Dies ist
+deshalb so wichtig, da wenn eine CPU zu wenig Rechenleistung besitzt, dann kann
+es hier zu einem Fehlverhalten der Dienste kommen. 
 \\textbf{Akzeptanzkriterien:}\\
 
 \begin{outline}
-  \1 Visualisierung der Auslastung der CPU Leistung für jede einzelne Instanz
-  \1 Alle Werte der CPU, wie soft-IRQ, wait oder interrupt müssen visualisiert
+  \1 Visualisierung der Auslastung und Verfügbarkeit der CPU Leistung für jede
```

Comment:
Vielleicht müssen wir hier auch nochmal etwas präziser werden. 
Angenommen ein Kern wird x3 virtualisiert, dann hat ein Kern 300 Prozent Leistung, kann aber in Wirklichkeit nur 100 Prozent ausführen. Wenn nun die drei VM's mit jeweils 1 virtuallen Kern (100 Prozent von den 300 Prozent) gleichzeitg die Leistung abrufen, dürfte jeder der VM's nur maximal 33 Prozent der Leistung bekommen, dann erhält jeder 1/3 des echten Kerns (weil ja x3 virtaulisiert).

Das heißt jedoch nicht, dass die VM's niemals auf 100 Prozent kommen können. Angenommen NUR einer der drei VM's greift auf die Leistung zu, dann könnte dieser (solange keine andere VM Leistung anfordert) die 100 Prozent voll nutzen. 

Das sollten wir bei dir hier klar erläutern.